bin/convert_data.py

```diff
@@ -66,16 +67,26 @@
                     # else:
                     #     tokens = np.concatenate([tokens, [0]])
 
-                    if annotations[i] == 'hit':
-                        i +=1
-                        continue
-                    elif annotations[i] == 'bounce':
+                    # if annotations[i] == 'hit':
+                    #     i +=1
+                    #     continue
+                    # elif annotations[i] == 'bounce':
+                    #     tokens = np.concatenate([tokens, [1]])
+                    # else:
+                    #     if random.randint(0,11) == 0:
+                    #         i +=1
+                    #         continue 
+                    #     tokens = np.concatenate([tokens, [0]])
```

Comment:
Do we need all this commented code for quick access or something?

Reply:
Yeah I kept all that there in case anyone else wanted to experiment with different labels.  I can remove if you want

Reply:
If there's a purpose to it, it's fine to leave. Same with below.